Detectors/FIT/common/calibration/include/FITCalibration/FITCalibrationApi.h

```diff
@@ -0,0 +1,94 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_FITCALIBRATIONAPI_H
+#define O2_FITCALIBRATIONAPI_H
+
+#include <utility>
+
+#include "CCDB/CcdbApi.h"
+#include "TH1F.h"
+#include "TGraph.h"
+#include "Rtypes.h"
+
+
+namespace o2::fit
+{
+
+
+template <typename CalibrationObjectType>
+class FITCalibrationApi final
```

Comment:
What is the advantage of having yet another api on top of the CcdbApi? Moreover, that the objects will be anyway provided by the framework?
In any case, if you need custom loading of objects, I would strongly suggest to use BasicCCDBManager, which supports object caching.

Reply:
Ok, I will try to use `BasicCCDBManager` instead of my api class.